src/main/java/org/elasticsearch/index/query/CommonTermsQueryBuilder.java

```diff
@@ -161,14 +227,18 @@ public CommonTermsQueryBuilder queryName(String queryName) {
         return this;
     }
 
+    public String queryName() {
+        return this.queryName;
+    }
+
     @Override
     public void doXContent(XContentBuilder builder, Params params) throws IOException {
-        builder.startObject(NAME);
-        builder.startObject(name);
+        builder.startObject(CommonTermsQueryParser.NAME);
```

Comment:
I think you have to update your branch, this constant should be part of the builder now, `builder.startObject(NAME);` should be ok I think